src/GotDownloader.ts

```diff
@@ -1,20 +1,46 @@
 import * as fs from 'fs-extra';
 import * as got from 'got';
 import * as path from 'path';
+import * as ProgressBar from 'progress';
 
 import { Downloader } from './Downloader';
 
 export class GotDownloader implements Downloader<any> {
   /**
    * @param options - see [`got#options`](https://github.com/sindresorhus/got#options) for possible keys/values.
+   * Additional options:
+   * * `quiet`: if `true`, disables the console progress bar (the `ELECTRON_GET_NO_PROGRESS`
+   *   environment variable also does this)
+   * * `getProgressCallback`: if defined, triggers every time `got`'s `downloadProgress` event
+   *   callback is triggered.
    */
   async download(url: string, targetFilePath: string, options?: any) {
```

Comment:
nit: Can we add types for options interface?

Reply:
Done in https://github.com/electron/get/pull/148/commits/eb21c794fff05cff9251a26c215da710446572a5